alacritty/src/display/window.rs

```diff
@@ -299,10 +301,12 @@ impl Window {
     #[cfg(windows)]
     pub fn get_platform_window(_: &Identity, window_config: &WindowConfig) -> WindowAttributes {
         let icon = winit::window::Icon::from_resource(IDI_ICON, None);
+        let taskbar_icon = winit::window::Icon::from_resource(IDI_ICON, None);
 
         WinitWindow::default_attributes()
             .with_decorations(window_config.decorations != Decorations::None)
             .with_window_icon(icon.ok())
```

Comment:
`icon.as_ref().ok().cloned()` here and pass `icon.ok()` to the `.with_taskbar_icon`.